### PR TITLE
Feature/24/link to profile

### PR DESCRIPTION
**Pull request by elishkaflint**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -8,6 +8,8 @@
   <% end %>
 </div>
 
+<h2><%= @user.first_name %>'s profile%></h2>
```
Comment (neobay991) [inline app/views/users/show.html.erb:11]: remove trailing %> after profile

File: spec/features/user_can_view_other_user_profiles_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.feature "Edit Posts", type: :feature do
```
Comment (neobay991) [inline spec/features/user_can_view_other_user_profiles_spec.rb]: Update test title

File: spec/features/user_can_view_other_user_profiles_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  before(:each) do
+    successful_sign_up
+    click_button "Log Out"
+    first_user_post
+    click_button "Log Out"
+    successful_sign_up_second_user
+    click_button "Log Out"
+    second_user_post
+  end
```
Comment (neobay991) [inline spec/features/user_can_view_other_user_profiles_spec.rb:13]: the before steps don't look right, I think we only need successful_sign_up or successful_sign_in ?

**Review by neobay991 — CHANGES_REQUESTED** [review]

File: app/views/users/show.html.erb (hunk 1 above)
Comment (elishkaflint) [reply app/views/users/show.html.erb:11]: thanks, caught this earlier in the profile picture branch so will update when we pull that work in

File: spec/features/user_can_view_other_user_profiles_spec.rb (hunk 2 above)
Comment (elishkaflint) [reply spec/features/user_can_view_other_user_profiles_spec.rb]: Thanks - updated

File: spec/features/user_can_view_other_user_profiles_spec.rb (hunk 3 above)
Comment (elishkaflint) [reply spec/features/user_can_view_other_user_profiles_spec.rb:13]: Noted. As discussed, web helpers need significant refactoring but we've taken decision as a team to focus on delivering priority features in the last sprint. Will add this decision point to our learning documentation.

Comment (neobay991) [reply spec/features/user_can_view_other_user_profiles_spec.rb:13]: Thanks!

File: app/views/users/show.html.erb (hunk 1 above)
Comment (neobay991) [reply app/views/users/show.html.erb:11]: 👍

**Review by neobay991 — APPROVED** [review]

**Review by neobay991 — DISMISSED** [review]
🐤

**Review by neobay991 — APPROVED** [review]
🐤

**Pull request merged**